hubspot/plugin.py

```diff
@@ -299,16 +310,16 @@ def _parse_hubspot_contact(api_item: dict, properties_schema: _HubSpotProperties
         additional_properties=_coerce_properties_to_lutra(
             {key: val for key, val in properties.items() if val is not None},
             schema=properties_schema,
-        )
+        ),
     )
 
 
 async def _list_contacts(
     schema: _HubSpotPropertiesSchema,
     pagination_token: Optional[HubSpotPaginationToken] = None,
-) -> Tuple[Sequence[HubSpotContact], Optional[HubSpotPaginationToken]]:
+) -> Tuple[List[HubSpotContact], Optional[HubSpotPaginationToken]]:
```

Comment:
Under what circumstance do you use Sequence vs List. I have gotten feedback that we should use Sequence and now we seem to be using List. Is there a consistency in how we are thinking about this?

Reply:
Generally I think we're supposed to use the more 'abstract' types for inputs when the implementation does not assume a particular concrete type, so that it can accept lists, tuples, etc when it doesn't matter for the implementation. When the implementation returns a specific concrete type (in this case, List) and we don't think the implementation would return a more generic type in the future, it's better to use the specific type so callers know what object they are getting.  The exception might be a public API where the implementation is currently a list but we want the signature to be specifically Sequence for generality / future-proofing (this would require callers to explicitly convert to list though, even if it's already a list).

The reason I made this change is that hubspot_search_contacts does return List[...] in its output signature, and so _list_contacts needs to as well, unless we wrapped the return in list(). Since this is an internal API, it seemed fine to be specific.  

We do return List[] output types for some of the public APIs here -- if we wanted to make that Sequence for future-proofing reasons, it'd be a change we'd need to make backwards compatible via versioning -- but we may want to do that sooner rather than later.  I'd batch that up with another breaking change though, to reduce number of versions.